src/java_bytecode/library/src/java/lang/Class.java

```diff
@@ -0,0 +1,423 @@
+/*
+ * Copyright (c) 1994, 2013, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package java.lang;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.cprover.CProver;
+import sun.reflect.CallerSensitive;
+import sun.reflect.Reflection;
+
+public final class Class<T> {
+
+    private Class() {}
+
+    private transient String name;
+
+    // TODO: these boolean fields model the internal encoding of classes
+    // they should be set by the getClass methods of the different classes
```

Comment:
This looks like some non-original comment. I think it would be good to have a very specific commenting format (as we did with the `DIFFBLUE MODEL LIBRARY` comments. It's important to be able to distinguish what is original from the JDK, and what has been amended to suit jbmc.

Reply:
This comment is present (without modification) in the same file in the model library, see `model/src/main/java/java/lang/Class.java` there.

Updating it here would break the sync between this file and what you have in the model library.

Are you sure you want us to do that?